docker-compose.yml

```diff
@@ -64,6 +64,73 @@ services:
                 '
         depends_on:
             - article-store-postgres
+    article-store-airflow-flower:
+        build:
+          context: ./workflow/airflow_libero
+        depends_on:
+            - article-store-airflow-redis
+        environment:
+            - EXECUTOR=Celery
+        ports:
+            - "5555:5555"
+        command: flower
+    article-store-airflow-postgres:
+        image: postgres:9.6
+        environment:
+            - POSTGRES_USER=airflow
+            - POSTGRES_PASSWORD=airflow
+            - POSTGRES_DB=airflow
```

Comment:
Does Airflow have these defaults?

Reply:
Yes, set [here](https://github.com/libero/walking-skeleton/blob/3c38fbc62638825dd9ea8a60386fe3564824651b/workflow/airflow_libero/script/entrypoint.sh#L11), so can be changed if need be really. Was just following some example deployments.